Plugin/Trajectory.cs

```diff
@@ -63,7 +63,9 @@ public struct Point
         public class Patch
         {
             public VesselState startingState { get; set; }
-            public double endTime { get; set; }
+            private double _endTime;

```

Comment:
endtime2? is nullable, again, this to match the other code.  I didn't want to make major changes by adding (double) to every reference for endTime, but that would also be a solution.